playbooks/PnP_Workflow_Manager_Playbook.yml

```diff
@@ -0,0 +1,111 @@
+      dnac_verify: "{{ dnac_verify }}"
+      dnac_port: "{{ dnac_port }}"
+      dnac_version: "{{ dnac_version }}"
+      dnac_debug: "{{ dnac_debug }}"
+      dnac_log_level: DEBUG
+
+  tasks:
+
+    - name: Import devices in bulk
+      cisco.dnac.pnp_workflow_manager:
+        <<: *dnac_login
+        dnac_log: True
+        state: merged
+        config_verify: True
+        config:
+            - device_info:
+              - serial_number: QD2425L8M7
+                state: Unclaimed
+                pid: c9300-24P
+                is_sudi_required: False
+
+              - serial_number: QTC2320E0H9
+                state: Unclaimed
+                pid: c9300-24P
+                hostname: Test-123
+
+              - serial_number: ETC2320E0HB
+                state: Unclaimed
+                pid: c9300-24P
+
+    - name: Add a new device and claim it
+      cisco.dnac.pnp_workflow_manager:
+        <<: *dnac_login
+        dnac_log: True
+        state: merged
+        config:
+          - site_name: Global/USA/San Francisco/BGL_18
+            device_info:
+              - serial_number: FJC2330E0BB
+                hostname: Test-9300-10
+                state: Unclaimed
+                pid: c9300-24P
+                is_sudi_required: True
+
+    - name: Claim an added Switch with template and image upgrade to a site only
+      cisco.dnac.pnp_workflow_manager:
+        <<: *dnac_login
+        dnac_log: True
+        state: merged
+        config:
+          - site_name: Global/USA/San Francisco/BGL_18
+            template_name: "Ansible_PNP_Switch"
+            image_name: cat9k_iosxe_npe.17.03.07.SPA.bin
+            project_name: Onboarding Configuration
+            template_details:
+              hostname: SJC-Switch-1
+              interface: TwoGigabitEthernet1/0/2
+            device_info:
+              - serial_number: FJC271924EQ
+                hostname: Switch
+                state: Unclaimed
+                pid: C9300-48UXM
+
+    - name: Claim an added Wireless Controller with template and image upgrade to a site only
+      cisco.dnac.pnp_workflow_manager:
+        <<: *dnac_login
+        dnac_log: True
+        state: merged
+        config:
+          - site_name: Global/USA/San Francisco/BGL_18
+            pnp_type: CatalystWLC
+            template_name: "Ansible_PNP_WLC"
+            image_name: C9800-40-universalk9_wlc.17.12.01.SPA.bin
+            template_params:
+              hostname: IAC-EWLC-Claimed
+            device_info:
+              - serial_number: FOX2639PAY7
+                hostname: New_WLC
+                state: Unclaimed
+                pid: C9800-CL-K9
+            gateway: 204.192.101.1
+            ip_interface_name: TenGigabitEthernet0/0/0
+            static_ip: 204.192.101.10
+            subnet_mask: 255.255.255.0
+            vlan_id: 1101
+
+    - name: Delete multiple devices from the Pnp dashboard #If device is not present it won't fail
```

Comment:
- name: Remove multiple devices from the PnP dashboard safely (ignores non-existent devices)

Reply:
Done